### ISSUE TITLE
Bug - only one display is shown if they have the same model/serial number

### ISSUE THREAD

**Issue by peterhoeg** [davidhi7/ddcci-plasmoid#44]
I have 2 relatively cheap 4k panels of the same model, but I am only shown 1 monitor for which to adjust brightness.

![ddcci](https://github.com/davidhi7/ddcci-plasmoid/assets/722550/79078a7b-6f8b-450d-a553-9618bac42f3f)

I'm guessing (without having looked at the code) this has to do with the fact that they have the same serial number.

<pre>
Display 1
   I2C bus:  /dev/i2c-6
   DRM connector:           card0-DP-1
   EDID synopsis:
      Mfg id:               HKC - UNK
      Model:                N07
      Product code:         10019  (0x2723)
      Serial number:        0000000000001
      Binary serial number: 0 (0x00000000)
      Manufacture year:     2023,  Week: 4
   VCP version:         2.1

Display 2
   I2C bus:  /dev/i2c-7
   DRM connector:           card0-DP-2
   EDID synopsis:
      Mfg id:               HKC - UNK
      Model:                N07
      Product code:         10019  (0x2723)
      Serial number:        0000000000001
      Binary serial number: 0 (0x00000000)
      Manufacture year:     2023,  Week: 4
   VCP version:         2.1

</pre>

**Comment by davidhi7** [comment]
Yes, monitors with identical serial numbers are sorted out as a workaround for an amdgpu driver bug. I think the only  solution to this is to add a setting that disables the search for duplicate 'identical' monitors altogether.

**Comment by ChristianAnke** [comment]
+1

**Comment by peterhoeg** [comment]
You cannot use the bus address or the connector for identification together with the serial?

**Comment by davidhi7** [comment]
The connector is also different, so (at least on my setup) the only way to detect duplicate monitors is by comparing the serial numbers.

**Comment by peterhoeg** [comment]
> The connector is also different, so (at least on my setup) the only way to detect duplicate monitors is by comparing the serial numbers.

Are you saying that in some cases a single monitor will show more than once connected to multiple outputs?

**Comment by davidhi7** [comment]
Comparing the i2c busses doesn't solve this issue. On my setup, the ddcutil detect output features identical serial numbers and different i2c busses as well, but the two entries do represent the identical monitor.

**Comment by peterhoeg** [comment]
The way I worked around this:

1. dump the EDID date from one monitor
2. edit the EDID file to make the following changes:
    1. change the serial from 1 to 2
    2. change the checksum to match
3. pass the modified EDID to the linux kernel on boot

And they would then appear properly as 2 different monitors due to the serials being different.

It was however short lived as it turns out one of the cheap non-brand monitors gave up on life, so I returned them both for a full refund and then simply bought LG instead.  That's what I get for trying to be cheap...

**Comment by davidhi7** [comment]
Sounds interesting, would happily include this in the README if this is a reliable and somewhat simple solution.
